### ISSUE TITLE
[a11y] Cannot export query to a label or report

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1251]
**To Reproduce**
1. Go to a query
2. Attempt to create a label or report based on it
3. See error

**Expected behavior**
It should create a new resource.

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/163607555-21fe8272-0f48-4402-acdd-892f1de1f416.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Edge
